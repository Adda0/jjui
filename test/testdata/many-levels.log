@  zo;zorupswq;.;true;false;false;true;some@email.com;2 weeks ago;main branch - single line
│ ○  t;topmxuzq;.;false;false;false;true;some@email.com;2 weeks ago;branch 2
├─╯
│ ○  v;vmylrovy;.;false;false;false;true;some@email.com;2 weeks ago;branch 13 - multiline description
│ │ ○  r;rqxpwkmr;.;false;false;false;true;some@email.com;2 weeks ago;branch 12
│ ├─╯
│ │ ○  wr;wrvqymln;.;false;false;false;true;some@email.com;2 weeks ago;branch 11
│ ├─╯
│ ○  q;qzlpqvqu;.;false;false;false;true;some@email.com;2 weeks ago;branch 1
├─╯
○  wl;wlqtxrzo;.;false;false;false;true;some@email.com;2 weeks ago;initial commit
◆  zz;zzzzzzzz;.;false;true;false;true;55 years ago