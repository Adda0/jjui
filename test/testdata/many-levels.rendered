○  zo
│ ○  t
├─╯
│ ○  v
│ │ ○  r
│ ├─╯
│ │ ○  wr
│ ├─╯
│ ○  q
├─╯
○  wl
○  zz
