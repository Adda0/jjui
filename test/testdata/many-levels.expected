@  zorupswq
│  (empty) main branch - single line
│ ○  topmxuzq
├─╯  (empty) branch 2
│ ○  vmylrovy
│ │  (empty) branch 13 - multiline description
│ │ ○  rqxpwkmr
│ ├─╯  (empty) branch 12
│ │ ○  wrvqymln
│ ├─╯  (empty) branch 11
│ ○  qzlpqvqu
├─╯  (empty) branch 1
○  wlqtxrzo
│  (empty) initial commit
◆  zzzzzzzz root()